api/execution/gen.go

```diff
@@ -1,3 +1,3 @@
 package execution
 
-//go:generate protoc -I.:../..:../../../../../github.com/gogo/protobuf:/usr/local/include --gogoctrd_out=plugins=grpc,import_path=github.com/docker/containerd/api/execution,Mgogoproto/gogo.proto=github.com/gogo/protobuf/gogoproto,Mgoogle/protobuf/descriptor.proto=github.com/gogo/protobuf/protoc-gen-gogo/descriptor:. execution.proto
+//go:generate protoc -I.:${PWD}/vendor/github.com/gogo/protobuf:/usr/local/include --gogoctrd_out=plugins=grpc,import_path=github.com/docker/containerd/api/execution,Mgogoproto/gogo.proto=github.com/gogo/protobuf/gogoproto,Mgoogle/protobuf/descriptor.proto=github.com/gogo/protobuf/protoc-gen-gogo/descriptor:. execution.proto
```

Comment:
So, shell doesn't get evaluated in `go:generate`.

This needs to be

`-I.:../../vendor:../../vendor/github.com/gogo/protobuf:../../../../../..:/usr/local/include`

This creates the following:

1. Local path import.
2. Vendor path for project.
3. `gogo/protobuf` as a top-level path space (i think this is right).
4. The current GOPATH as a path space.
5. `/usr/local/include` as a path space.

I know this is weird but it keeps the import paths for the individual protobuf files from becoming non-canonical when used as dependencies.